packages/cx-elements/doc/CX_SiteTypography.md

```diff
@@ -1,100 +1,115 @@
 # CanvasX Site Typography Setup/Override
 
-The site typography consists of updating fonts, colors, font sizes and very elemental tokens.
+The site typography consists of updating fonts, colors, font sizes, and very elemental tokens.
```

Comment:
@hvanyo, wasn't sure what "very elemental tokens" was intended to be, so I didn't touch it. :)

Reply:
@misterjones  "very elemental tokens" - like h1, h2, h3